package.json

```diff
@@ -38,7 +39,9 @@
     "nyc": "^11.8.0",
     "stylelint": "^9.2.0",
     "stylelint-config-standard": "^18.2.0",
-    "tape": "^4.9.0"
+    "tape": "^4.9.1",
+    "tape-async": "^2.3.0",
+    "tape-promise": "^3.0.0"
```

Comment:
Do we need **BOTH** `tape-async` and `tape-promise`? I think the latter is currently unused.

Reply:
Good catch.